### PR TITLE
WIP: Update "What" page

### PR DESCRIPTION
**Pull request by awgreene**

### CODE REVIEW

File: content/what.html [hunk 1]
```diff
@@ -19,27 +19,27 @@ <h2 class="of-heading of-heading--md">What is an Operator after all?</h2>
...
+        <p class="of-section--largetext__content">Cluster Admins, Software, and DevOps engineers work together to create an operator that encapsulates their shared knowledge, offering a kubernetes native way of managing the application.</p>
         <a href="https://sdk.operatorframework.io/" class="of-button of-button--tertiary">Go to Operator SDK
           <svg class="of-button__icon" enable-background="new 0 0 22 22" version="1.1" viewBox="0 0 22 22" xml:space="preserve" xmlns="http://www.w3.org/2000/svg">
               <path d="M19.5,8l-7.3-7.6c-0.4-0.4-1.1-0.4-1.5,0L9.3,1.8c-0.4,0.4-0.4,1.1,0,1.6l5.2,5.5H1.1C0.5,8.9,0,9.3,0,10V12  c0,0.6,0.5,1.1,1.1,1.1h13.5l-5.2,5.5c-0.4,0.4-0.4,1.1,0,1.6l1.4,1.5c0.4,0.4,1.1,0.4,1.5,0l9.4-9.9c0.4-0.4,0.4-1.1,0-1.6L19.5,8z  "/>
               </svg>
         </a>
-        <p class="large-text"><span>Persona, </span>
-        <span>persona, </span>
-        <span>persona.</span></p>
+        <p class="large-text"><span>Software Engineers, </span>
```
Comment (asmacdo) [inline content/what.html:28]: might have to use a different class, its doing something funky 
Screenshot from 2020-11-17 15-50-12

**Review by asmacdo — APPROVED** [review]
Already a huge improvement. Once the "persona persona persona" replacement text renders correctly, I think this can merge.

File: content/what.html [hunk 2]
```diff
@@ -19,27 +19,27 @@ <h2 class="of-heading of-heading--md">What is an Operator after all?</h2>
...
       <a href="https://www.operatorhub.io" class="of-button of-button--tertiary">Go to OperatorHub.io
           <svg class="of-button__icon" enable-background="new 0 0 22 22" version="1.1" viewBox="0 0 22 22" xml:space="preserve" xmlns="http://www.w3.org/2000/svg">
               <path d="M19.5,8l-7.3-7.6c-0.4-0.4-1.1-0.4-1.5,0L9.3,1.8c-0.4,0.4-0.4,1.1,0,1.6l5.2,5.5H1.1C0.5,8.9,0,9.3,0,10V12  c0,0.6,0.5,1.1,1.1,1.1h13.5l-5.2,5.5c-0.4,0.4-0.4,1.1,0,1.6l1.4,1.5c0.4,0.4,1.1,0.4,1.5,0l9.4-9.9c0.4-0.4,0.4-1.1,0-1.6L19.5,8z  "/>
               </svg>
       </a>
-      <p class="large-text"><span>Persona, </span>
-        <span>persona, </span>
-        <span>persona.</span></p></div>
+      <p class="large-text"><span>Cluster Admins, </span>
+        <span>Kubernetes Users, </span>
```
Comment (dmesser) [inline content/what.html:41]: Wouldn't OperatorHub.io also appeal to Operator Developers to publish their work?

File: content/what.html [hunk 3]
```diff
@@ -19,27 +19,27 @@ <h2 class="of-heading of-heading--md">What is an Operator after all?</h2>
...
           <svg class="of-button__icon" enable-background="new 0 0 22 22" version="1.1" viewBox="0 0 22 22" xml:space="preserve" xmlns="http://www.w3.org/2000/svg">
               <path d="M19.5,8l-7.3-7.6c-0.4-0.4-1.1-0.4-1.5,0L9.3,1.8c-0.4,0.4-0.4,1.1,0,1.6l5.2,5.5H1.1C0.5,8.9,0,9.3,0,10V12  c0,0.6,0.5,1.1,1.1,1.1h13.5l-5.2,5.5c-0.4,0.4-0.4,1.1,0,1.6l1.4,1.5c0.4,0.4,1.1,0.4,1.5,0l9.4-9.9c0.4-0.4,0.4-1.1,0-1.6L19.5,8z  "/>
               </svg>
         </a>
-        <p class="large-text"><span>Persona, </span>
-        <span>persona, </span>
-        <span>persona.</span></p>
+        <p class="large-text"><span>Software Engineers, </span>
+        <span>DevOps Engineers, </span>
+        <span>Cluster Admins.</span></p>
```
Comment (dmesser) [inline content/what.html:30]: I wonder if we can convey here that the best people to write an Operators are the owners / developer of said application and SRE folks. WDYT?

**Review by dmesser — CHANGES_REQUESTED** [review]

File: content/what.html (hunk 2 above)
Comment (awgreene) [reply content/what.html]: Good idea! I've updated the doc!

File: content/what.html (hunk 3 above)
Comment (awgreene) [reply content/what.html]: I think the updated commit makes this point clearer as well!

File: content/what.html (hunk 1 above)
Comment (awgreene) [reply content/what.html]: I've had to revert to one-word names of personas, please review.

File: content/what.html [hunk 4]
```diff
@@ -19,27 +19,27 @@ <h2 class="of-heading of-heading--md">What is an Operator after all?</h2>
...
           <svg class="of-button__icon" enable-background="new 0 0 22 22" version="1.1" viewBox="0 0 22 22" xml:space="preserve" xmlns="http://www.w3.org/2000/svg">
               <path d="M19.5,8l-7.3-7.6c-0.4-0.4-1.1-0.4-1.5,0L9.3,1.8c-0.4,0.4-0.4,1.1,0,1.6l5.2,5.5H1.1C0.5,8.9,0,9.3,0,10V12  c0,0.6,0.5,1.1,1.1,1.1h13.5l-5.2,5.5c-0.4,0.4-0.4,1.1,0,1.6l1.4,1.5c0.4,0.4,1.1,0.4,1.5,0l9.4-9.9c0.4-0.4,0.4-1.1,0-1.6L19.5,8z  "/>
               </svg>
         </a>
-        <p class="large-text"><span>Persona, </span>
-        <span>persona, </span>
-        <span>persona.</span></p>
+        <p class="large-text"><span>Developers, </span>
+        <span>SREs, </span>
+        <span>Admins.</span></p>
```
Comment (asmacdo) [inline content/what.html:30]: :+1:

**Review by asmacdo — APPROVED** [review]

**Review by dmesser — APPROVED** [review]

**Pull request merged**